config/3.3/eventManager/templates/gobal-image-pull-secret.yaml

```diff
@@ -0,0 +1,112 @@
+{{- if .Values.internalbuild }}
```

Comment:
To keep the consistent naming convention, can we use `config/3.x/event-manager` instead of `config/3.x/eventManager`, and use `config/3.x/ai-manager` instead of `config/3.x/aiManager`?

Reply:
Will commit a patch to change this